fix_simulation/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Use Rocky Linux 9.3 as the base image and set platform to linux/amd64
+FROM --platform=linux/amd64 rockylinux:9.3
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Update the system
+RUN dnf -y update
+
+# Upgrade to Rocky Linux 9.4 (optional)
+RUN dnf -y install dnf-plugin-system-upgrade \
+    && dnf -y system-upgrade download --releasever=9.4
+
+# Install development tools and necessary dependencies
+RUN dnf -y group install "Development Tools"
+RUN dnf -y install epel-release
+RUN dnf -y install cmake clang clang-devel clang-tools-extra
```

Comment:
This includes a lot of packages, are they all needed?

Reply:
Removed unnecessary packages https://github.com/true-markets/tools/pull/1/commits/bc9f43f7cf1a6776d447238aa0e5b3075788ae7f . This has bare minimum to run the script with quickfix